nsqadmin/static_debug.go

```diff
@@ -1,19 +0,0 @@
-//go:build go1.16 && debug
-// +build go1.16,debug
-
-package nsqadmin
-
-import (
-	"io/ioutil"
-	"os"
-	"path"
-)
-
-func staticAsset(name string) ([]byte, error) {
-	path := path.Join("../../nsqadmin/static/build", name)
```

Comment:
I think the ugliness and problems likely resulting from this hard-coded path justify this change

Reply:
indeed, this was the motivation :)
thanks